lib/ansible/plugins/connection/network_cli.py

```diff
@@ -395,6 +395,10 @@ def receive(self, command=None, prompts=None, answer=None, newline=True):
             window = self._strip(recv.read())
             if prompts and not handled:
                 handled = self._handle_prompt(window, prompts, answer, newline)
+            elif prompts and handled:
+                # check again even when handled, a sub-prompt could be
+                # repeating (like in the case of a wrong enable password, etc)
+                self._handle_prompt(window, prompts, answer, newline)
```

Comment:
I didn't get the purpose of this call again can you please share a reference example. 
For wrong enable password it will again end up sending the wrong password?
Also, multiple prompts can be given as a list in the input.